controller.py

```diff
@@ -814,12 +754,9 @@ def base_url(full_url):
         return make_response(content, status_code, headers)
 
 
-class URNLookupController(CoreURNLookupController, ISBNEntryMixin):
+class URNLookupController(CoreURNLookupController):
 
-    UNRESOLVABLE_IDENTIFIER = "I can't gather information about an identifier of this type."
-    IDENTIFIER_REGISTERED = "You're the first one to ask about this identifier. I'll try to find out about it."
-    WORKING_TO_RESOLVE_IDENTIFIER = "I'm working to locate a source for this identifier."
-    SUCCESS_DID_NOT_RESULT_IN_PRESENTATION_READY_WORK = "Something's wrong. I have a record of covering this identifier but there's no presentation-ready work to show you."
+    WORKING_TO_RESOLVE_IDENTIFIER = "I don't have enough information about this Identifier yet.\nDetailed work log:\n "
```

Comment:
This architecture dramatically cuts down on the possible `/lookup` responses -- either there's a presentation-ready work or there isn't. If there isn't, we provide a record of what work is registered for the given Identifier. If there is, there might be more work to be done later, but at least we have some information about the Identifier.

Reply:
If there is more work to be done later, the client won't know? Or will it know by the absence of some of the information it wanted?

Reply:
The client won't know, but there's nothing it could do differently if it did know.
Overall we need better visibility into where the bottlenecks and failure points are in coverage, but on the level of individual items I think most people won't care once we get the basics -- cover image and description.